### PR TITLE
[PL-47879] PL 1.27.10 Release Note

### PR DESCRIPTION
**Pull request by saifeemustafaqharness**
[PL-47879](https://harness.atlassian.net/browse/PL-47879)

https://harness.atlassian.net/wiki/spaces/PD/pages/21673181249/NG+Release+SAAS+February+26th+2024+for+1.27.x-+Build

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screenshot. 

[PL-47879]: https://harness.atlassian.net/browse/PL-47879?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by saifeemustafaqharness** [comment]
@pratmit @brian-f-harness @nataraja2018 Ready for Review 1.27.10

File: release-notes/platform.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Platform release notes
 sidebar_label: Platform
 tags: [NextGen, "platform"]
-date: 2024-02-27:T10:00:30
+date: 2024-03-04:T10:00:30
```
Comment (brian-f-harness) [inline release-notes/platform.md:5]:
```suggestion
date: 2024-03-05:T10:00:30
```

File: release-notes/platform.md [hunk 2]
```diff
@@ -82,6 +82,22 @@ The following deprecated API endpoints are longer supported:
 
 ## February 2024
 
+### Version 1.27.10<!--  March 4, 2024 -->
```
Comment (brian-f-harness) [inline release-notes/platform.md:85]:
```suggestion
### Version 1.27.10<!--  March 5, 2024 -->
```

File: release-notes/platform.md [hunk 3]
```diff
@@ -82,6 +82,22 @@ The following deprecated API endpoints are longer supported:
...
+
+#### New features and enhancements
+
+- Support added to enable OPA policy for TTL enforcement while creating or updating an API Key Token. (PL-46779)
```
Comment (brian-f-harness) [inline release-notes/platform.md:89]: Are they any docs affected by this change?

**Review by brian-f-harness — APPROVED** [review]

**Pull request merged**